src/main/java/net/dv8tion/jda/api/events/thread/member/GenericThreadMemberEvent.java

```diff
@@ -39,22 +47,42 @@ public GenericThreadMemberEvent(@Nonnull JDA api, long responseNumber, ThreadCha
         this.threadMember = threadMember;
     }
 
+    /**
+     * The Snowflake id of the interacting {@link ThreadMember}. This is unique to every entity and will never change.
+     *
+     * @return Never-null String containing the Id.
+     */
     public String getThreadMemberId()
     {
         return Long.toUnsignedString(getThreadMemberIdLong());
     }
 
+    /**
+     * The Snowflake id of the interacting {@link ThreadMember}. This is unique to every entity and will never change.
+     *
+     * @return Long containing the Id.
+     */
     public long getThreadMemberIdLong()
     {
         return threadMemberId;
     }
 
+    /**
+     * The interacting {@link ThreadMember}.
+     *
+     * @return The interacting {@link ThreadMember}.
+     */
     @Nullable
     public ThreadMember getThreadMember()
     {
         return threadMember;
     }
 
+    /**
+     * The interacting {@link ThreadMember} as a {@link Member GuildMember}.
```

Comment:
```suggestion
     * The interacting {@link ThreadMember} as a {@link Member}.
```

Reply:
I explicitly called it "guild member" for  clarification

Reply:
Yeah, its less clear in my opinion as it looks like a different type. Maybe do `guild {@link Member}` instead?